modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -612,18 +612,44 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       throw new Error('Invalid user key - missing backupNShare');
     }
 
+    let signablePayload;
+    let derivationPath;
+
+    if (requestType === RequestType.tx) {
+      assert(
+        txRequestResolved.transactions || txRequestResolved.unsignedTxs,
+        'Unable to find transactions in txRequest'
+      );
+      const unsignedTx =
+        txRequestResolved.apiVersion === 'full'
+          ? txRequestResolved.transactions![0].unsignedTx
+          : txRequestResolved.unsignedTxs[0];
+      signablePayload = Buffer.from(unsignedTx.signableHex, 'hex');
+      derivationPath = txRequestResolved.transactions![0].unsignedTx.derivationPath;
+    } else if (requestType === RequestType.message) {
+      signablePayload = (params as TSSParamsForMessage).bufferToSign;
+      // TODO Message singing with derivation path
```

Comment:
@jeanlarrieux  can you take look at what it takes to add derivation path to message signing?